src/lib/Form/Data/ContentType/DraftContentTypesDeleteData.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\ContentType;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+
+class DraftContentTypesDeleteData
+{
+    /** @var ContentType[]|null */
```

Comment:
This is a lie 🙈  https://github.com/ezsystems/ezplatform-admin-ui/pull/1377/files#diff-f0347094fc4c333bea71c35981fcc69aR640 tells me this is an `array<int, bool>`